meta/vqtl2

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "vqtl2",
+  "author": "Robert W. Corty",
+  "license": "GPL3",
+  "raster": "http://hexb.in/hexagons/vqtl2.png",
+  "vector": "http://hexb.in/vector/vqtl2.svg",
+  "description": "R package for genetic mapping",
+  "order_online_url": "this is optional. should be a link to where people can buy the sticker online"
```

Comment:
Can you remove this line if you aren't filling it out? I think at the very least it should point to a valid URL if it's in here. (Also take care to remove the trailing comma in the next line if you do so.)